ui-lib/src/main/java/co/electriccoin/zcash/global/DeepLinkUtil.kt

```diff
@@ -0,0 +1,46 @@
+package co.electriccoin.zcash.global
+
+import android.net.Uri
+import android.text.TextUtils
+import android.util.Base64
+import cash.z.ecc.android.sdk.ext.convertZecToZatoshi
+import cash.z.ecc.android.sdk.model.Zatoshi
+import co.electriccoin.zcash.spackle.Twig
+import co.electriccoin.zcash.ui.common.AMOUNT_QUERY
+import co.electriccoin.zcash.ui.common.MEMO_QUERY
+import co.electriccoin.zcash.ui.common.ZEC_MAX_AMOUNT
+
+object DeepLinkUtil {
+
+    fun getSendDeepLinkData(uri: Uri): SendDeepLinkData? {
+        // sample deep link: zcash:zs1j29m7zdhhyy2eqrz89l4zhk0angqjh368gqkj2vgdyqmeuultteny36n3qsm47zn8du5sw3ts7f?amount=0.001&memo=c2RrZmp3cw
```

Comment:
ZIP-321 is updated to include support for Unified Address and multiple addresses. Check implementation against the spec.

Reply:
Next PR

Reply:
Sure, will make this changes in next PR